airbyte-cdk/python/airbyte_cdk/sources/declarative/requesters/paginators/strategies/pagination_strategy.py

```diff
@@ -15,6 +15,14 @@ class PaginationStrategy:
     Defines how to get the next page token
     """
 
+    @property
+    def initial_token(self) -> Optional[Any]:
+        """
+        Return the initial value of the token
+        """
+
+        return None
```

Comment:
nit: I would prefer we leave this as abstract and force the child classes to define it since it'll force us to think about the initial case.

Concretely, this just means implementing it in the cursor pagination strategy